projects.html

```diff
@@ -1,9 +1,58 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Projects</title>
+ <title>Projects</title>
+ <link rel="stylesheet" href="css/projects.css"/>
 </head>
 <body>
-
+	<!-- IMAGES -->
+    <img src="img/p1.png" class="img1">
+	<img src="img/p2.png" class="img2">
+	<img src="img/p3.jpg" class="img3">
+	<img src="img/p4.jpg" class="img4">
+	<img src="img/p5.jpg" class="img5">
+	<!-- BUTTONS -->
+	<div class="button button1" >
+	 <a href="https://github.com/DSC-RSU/DSC-RSU-Website">
+	  <font color="black">Link To Project</font>
+	  </a>
+	  </div>
+	<div class="button button2" >
+	 <a href="">
+	  <font color="navy">Link To Project</font> 
+	  </a>
+	  </div>
+	<div class="button button3">
+	 <a href="">
+	  <font color="white" >Link To Project</font>
+	  </a>
+      </div>
+	<div class="button button4">
+	 <a href="">
+	  Link To Project 
+	  </a>
+	  </div>
+	<div class="button button5">
+	 <a href="">
+	  <font color="white"></font>Link To Project</font> 
+	  </a>
+	  </div>
+	<!-- TEXTS -->
+     <div class="t1">
+	  <font color="black">Status:</font><span class="glow"> <font color=#BA4A00>Ongoing</font></span>
+	  </div>
+      </div> 
+	 <div class="t2">
+	  <font color="black">Status:</font> <font color="red">Not yet Started</font>
```

Comment:
Put these statuses in center below each image like this
![image](https://user-images.githubusercontent.com/46588494/94989479-cb5a7800-0592-11eb-900f-bf93c582773c.png)
Align them in 4 projects in a row

Reply:
I added projects in 2 columns , you mean like i should added projects in 4 columns...

Reply:
In this picture you can see 3 in a row and 2 columns. You have to add 4 projects in a row and 2 columns. Also change the position of Status: ongoing from top of the picture to bottom as seen in this picture.

Reply:
won't the projects look more shorter

Reply:
> won't the projects look more shorter

shorter you mean to say? clumsy? if that seems so put it into 3 then

Reply:
okk